testdata/p4_16_samples_outputs/loop3-err.p4-stderr

```diff
@@ -0,0 +1,4 @@
+loop3-err.p4(51):syntax error, unexpected IN
```

Comment:
This error message is confusing, maybe we can improve it? 
@ChrisDodd do you know how easy that is?